cli/cmdlineparser.cpp

```diff
@@ -570,9 +575,12 @@ CmdLineParser::Result CmdLineParser::parseFromArgs(int argc, const char* const a
                     mSettings.addEnabled("performance");
                     mSettings.addEnabled("portability");
                 }
-                if (enable_arg.find("information") != std::string::npos) {
+                if (enable_arg.find("information") != std::string::npos && logMissingInclude == 0) {
+                    ++logMissingInclude;
```

Comment:
I envision:

     if (enable_arg.find("information") != std::string::npos && !mSettings.isEnabled("missingInclude"))
          logMissingInclude = true;

Reply:
in this case if we had `--disable=missingInclude` before `--enable=information` we would trigger the log about missing include  which is not logical as we've already used it

Reply:
if `--disable=missingInclude`  is before `--enable=information` it does not have an effect though. The missingInclude will be enabled as far as I know.